spack.yaml

```diff
@@ -0,0 +1,15 @@
+spack:
+  specs:
+    - acts@21.1.0 +json
+    - detray@0.22.0
+    - covfie@0.5.0
+    - algebra-plugins +vecmem +vc +eigen
+    - vecmem +cuda
```

Comment:
Why no explicit version specification for vecmem? The very latest version may not always work. (In fact I plan to introduce a non-backwards compatible update soon.)